build/images/base-windows/README.md

```diff
@@ -0,0 +1,22 @@
+# images/base-windows
+
+This Docker image includes download libraries for building windows image.
+
+If you need to build a new version of the image and push it to Dockerhub, you
+can run the following:
+
+```bash
+cd build/images/base-windows
+WIN_BUILD_TAG= echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10
```

Comment:
Please add commands to assign values to this variables or have a separate make target to build base-windows image so anyone following the doc can get consistent result.

Reply:
Thanks, I have updated README commands. I can successfully build image with a43c77d755 tag on my test environment now.  @tnqn